embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/anr/detection/LivenessCheckScheduler.kt

```diff
@@ -59,7 +60,8 @@
     fun startMonitoringThread() {
         enforceThread(anrMonitorThread)
         if (!state.started.getAndSet(true)) {
-            logger.logInfo("Started heartbeats to capture ANRs.")
+            logger.logInfo("Start ANR detection...")
+            resetResponsivenessMonitors()
```

Comment:
This line will be removed in the next PR as it's no longer necessary to do this manual clean up when the bug where we were still pinging the thread in the background is fixed.